### ISSUE TITLE
includes folders should be ignored automatically

### ISSUE THREAD

**Issue by oscarotero** [lumeland/lume#408]
### Enter your suggestions in details:

Example:

```ts
site.use(sass({
  includes: "sass"
}));
```

The folder /sass should be ignored automatically without need to do `site.ignore("sass")`.

**Comment by patroytall** [comment]
Documentation for SASS plugin:

"By default, the plugin is configured to look into the _includes directory to find the imported files (you can change this behaviour by editing the includes option)"

I have tried putting files under "_includes" but they are not getting picked up. I did a bit of debugging and the processor when going through pages does not include files under "_include" thus the ".scss" files are not found.

Using a top level "sass" works if I do not use a custom includes like in this bug.

In summary, the SASS plugin includes seem broken. I am new to Lume so I can log a bug with confidence.

**Comment by oscarotero** [comment]
@patroytall Do you have any `.scss` file outside the `_includes` folder that imports these files? For example, let's say you have the following structure:

```
styles.scss
_includes/colors.scss
```

In your styles.scss file, you can have the following code:

```scss
@import colors.scss
```

and this import should be resolved to `_includes/colors.scss`.

**Comment by patroytall** [comment]
OK, thank you. So yes, to get it working I added a /styles outside the "_includes". I now understand that "_includes" is for imports. Documentation is correct as is.
